nodejs/express/books/ibmi_utils.js

```diff
@@ -0,0 +1,68 @@
+const odbc = require('odbc');
+
+let ibmi = (function() {
+    let module = {};
+
+    // cb format: err, result
+    module.runSql = function(conn, sql, cb) {
+        try {
```

Comment:
This is wrapped in a `try/catch` block, but the `query` function won't throw unless using Promises:

```
let result = await conn.query(sql);
```

Instead, all errors are returned on the callback function when using callbacks.

Reply:
Didn't know that! Removed try&catch